cloudigrade/api/clouds/aws/tasks/inspection.py

```diff
@@ -111,6 +112,30 @@ def _build_cloud_init_script(ami_id):
     houndigrade_image = (
         f"{settings.HOUNDIGRADE_ECS_IMAGE_NAME}:{settings.HOUNDIGRADE_ECS_IMAGE_TAG}"
     )
+    cw_config = {
+        "logs": {
+            "logs_collected": {
+                "files": {
+                    "collect_list": [
+                        {
+                            "file_path": "/var/log/houndigrade",
+                            "log_group_name": f"houndigrade-"
+                            f"{settings.CLOUDIGRADE_ENVIRONMENT}",
+                            "log_stream_name": f"{ami_id}-inspection",
+                            "retention_in_days": 14,
```

Comment:
Did you mean to leave this `14` hard-coded instead of configurable? I actually think 14 days is fine and doubt we'll change it, but whenever I see a magic number, I have to ask at least. 🙂

Reply:
Hmm, I did miss "Let's consider 90 days for retention/lifecycle/etc." from the original issue. I'll update it to be pulled from a settings env var with a default to 90.

Reply:
Change is in, let me know what you think. Thank you for the spot!